TwitchChannelPointsMiner/TwitchChannelPointsMiner.py

```diff
@@ -207,8 +207,6 @@ def run(self, streamers: list = [], blacklist: list = [], followers=False):
                 self.twitch.load_channel_points_context(streamer)
                 self.twitch.check_streamer_online(streamer)
                 self.twitch.viewer_is_mod(streamer)
```

Comment:
Do we keep the info the viewer is a mod or not?

It was used only to not cast predictions but could be useful someday.

Reply:
Just comment for the moment :)
We save one request :laughing:

Reply:
Feels weird to just comment that part. The field will still be present in the Streamer object but with a potentially wrong value. And if one day it is needed we may spend time to understand why it is wrong.

In my opinion would be better to let it as it is and have useless requests being made, or remove it all together.

Will still comment it for now.